test/functional/test_framework/test_framework.py

```diff
@@ -102,6 +102,7 @@ def __init__(self):
         self.rpc_timeout = 60  # Wait for up to 60 seconds for the RPC server to respond
         self.supports_cli = True
         self.bind_to_localhost_only = True
+        self.wallet_names = None
```

Comment:
in commit 5338dfaef077b5c38e3a87d76f12068147a5db2d:

Would be good to add a short comment here to say:

* Will be used to create wallets on setup
* Defaults to create a default wallet for each node
* List can be shorter than the number of nodes, or include None entries to skip wallet creation